### PR TITLE
upgrade to the latest typescript (^2.8.3)

### PR DESCRIPTION
**Pull request by marcinwadon**
#6

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +24,7 @@
   },
   "homepage": "https://github.com/gcanti/fp-ts-rxjs",
   "dependencies": {
-    "fp-ts": "^1.0.1",
+    "fp-ts": "^1.5.0",
```
Comment (gcanti) [inline package.json]: Upgrading `typescript` is ok, not sure why you want to change the supported versions of `fp-ts` though

File: package.json [hunk 2]
```diff
@@ -36,7 +35,7 @@
     "ts-node": "3.2.0",
     "tslint": "4.4.2",
     "tslint-config-standard": "4.0.0",
-    "typescript": "^2.7.1"
+    "typescript": "^2.8.3u"
```
Comment (gcanti) [inline package.json]: is this a typo?

**Review by gcanti — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (marcinwadon) [reply package.json]: Should it be the lowest supported version?

File: package.json (hunk 2 above)
Comment (marcinwadon) [reply package.json]: Yes, good catch. I saw that but I was sure I removed that :)

File: package.json (hunk 1 above)
Comment (gcanti) [reply package.json]: in "dependencies" yes

**Comment by marcinwadon** [comment]
@gcanti I've reverted `fp-ts`. What about rxjs? There is a new version which changes a lot but it's in dependencies (so it also should stay as low as possible?).

**Comment by gcanti** [comment]
The current version of `fp-ts-rxjs` (`0.4.0`) supports `rxjs@5.x.x`. 

We could add support for `rxjs@6.x.x` and then release a `0.5.0` version of `fp-ts-rxjs`.

**Review by gcanti — APPROVED** [review]

**Comment by gcanti** [comment]
> We could add support for rxjs@6.x.x

@marcinwadon would you like to send a PR for that?

**Comment by marcinwadon** [comment]
@gcanti I'll try to do it in about 6-7 hours

**Pull request merged**